distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -96,6 +96,12 @@ solr.commit.nrt.metacardTypes=workspace,metacard.query,metacard.list,query-templ
 solr.highlight.blacklist=metacard-tags
 solr.highlight.enabled=false
 
+# The number of characters to limit the text sorting fields to
```

Comment:
✏️ I am not sure if we need to include these properties here if the defaults are good for almost all deployments.